alpaca/rest.go

```diff
@@ -29,6 +29,7 @@ type Client interface {
 	GetClock() (*Clock, error)
 	GetCalendar(start, end *string) ([]CalendarDay, error)
 	ListOrders(status *string, until *time.Time, limit *int, nested *bool) ([]Order, error)
+	ListOrdersV2(req ListOrdersRequest) ([]Order, error)
```

Comment:
I don't like the naming. What do you think about `ListOrdersWithRequest`?

Reply:
That's fine with me.